src/components/composer/composer/composer.ts

```diff
@@ -356,6 +359,22 @@ export class Composer extends Component<Props, SpreadsheetChildEnv> {
     this.processContent();
   }
 
+  showAutocomplete(searchTerm: string) {
+    this.autoCompleteState.showProvider = true;
+    let values = Object.keys(functionRegistry.content).map((key) => {
+      return {
+        text: key,
+        description: functionRegistry.get(key).description,
+      };
+    });
+
+    values = values
+      .filter((t) => t.text.toUpperCase().startsWith(searchTerm.toUpperCase()))
```

Comment:
maybe a future IMP could be to use fuzzy look up since we have an implementation now